src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -322,9 +322,26 @@ impl<'a, 'tcx> DocFolder for LinkCollector<'a, 'tcx> {
                     continue;
                 }
 
+                // In order to correctly resolve intra-doc-links we need to
```

Comment:
Thanks a lot for this comment, this is very helpful!

Reply:
You're welcome :D